packages/repack/android/src/main/java/com/callstack/repack/ChunkManagerModuleImpl.kt

```diff
@@ -0,0 +1,91 @@
+package com.callstack.repack
+
+import android.os.Handler
+import com.facebook.react.bridge.*
+
+class ScriptManagerModuleImpl {
+    companion object {
+        const val NAME = "ScriptManager"
+    }
+
+    private fun runInBackground(fn: () -> Unit) {
+        val handler = Handler()
+        val runnable = Runnable {
+            fn()
+        }
+        handler.postDelayed(runnable, 0)
+
+    }
+
+    fun loadScript(scriptId: String, configMap: ReadableMap, promise: Promise, remoteLoader: RemoteScriptLoader, fileSystemLoader: FileSystemScriptLoader) {
```

Comment:
[nit]: as an improvement, we can consider rewriting it to follow the bob pattern: 
https://github.com/callstack/react-native-builder-bob/tree/main/packages/create-react-native-library/templates/java-library-mixed/android/src

cc @thymikee